airflow_password.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+airflow_password=$(aws --output text secretsmanager get-secret-value --secret-id airflow_password --query SecretString --region us-east-1)
+echo $airflow_password
```

Comment:
Is there a reason to store the output in a variable that's echoed? The following version should achieve the same. 

```suggestion
aws --output text secretsmanager get-secret-value --secret-id airflow_password --query SecretString --region us-east-1
```